test/_fakedata/_create_fake_data.py

```diff
@@ -35,6 +36,22 @@ def create_files(folder, suffix, data, encoding=False):
         archive.add(folder)
 
 
+def add_data_to_tar(archive, name, value):
```

Comment:
I'm not sure what you mean. The tar file for the test case, `wds.tar` is not empty. I added it to `test/_fakedata` and the repository because there are already other tar files used for test cases there as well. Of course, it could also be generated on the fly.